Mathlib/CategoryTheory/Abelian/GrothendieckAxioms.lean

```diff
@@ -0,0 +1,86 @@
+/-
+Copyright (c) 2023 Adam Topaz. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Isaac Hernando, Coleton Kotch, Adam Topaz
+-/
+
+import Mathlib.CategoryTheory.Limits.Filtered
+import Mathlib.CategoryTheory.Limits.Preserves.Finite
+
+/-!
+
+# Grothendieck Axioms
+
+This file defines some of the Grothendieck Axioms for abelian categories, and proves
+basic facts about them.
+
+## Definitions
+
+- `AB4` -- an abelian category satisfies `AB4` provided that coproducts are exact.
+- `AB5` -- an abelian category satisfies `AB5` provided that filtered colimits are exact.
+- The duals of the above definitions, called `AB4Star` and `AB5Star`.
```

Comment:
Could you add references (e.g. to the Stacks project) because these phrasing of the definitions are not exactly the same as in the original Grothendieck Tohoku article.

Reply:
@adamtopaz Did you have any specific references? [The stacks site](https://stacks.math.columbia.edu/tag/079A) is phrased in a way that presupposes the category to be Abelian, which we, for good reason, don't...

Reply:
I wasn't following any references. I suppose that in lieu of references you could explain in the module docstring how the definitions here relate to those in Tohoku and/or the stacks project.

Reply:
Are the few sentences sufficient, @joelriou?